controllers/prep_handlers.go

```diff
@@ -58,68 +149,69 @@ func (r *ImageBasedUpgradeReconciler) handlePrep(ctx context.Context, ibu *ranv1
 
 	progressfile := filepath.Join(utils.Path, "prep-progress")
 
-	_, err = os.Stat(progressfile)
+	_, err = os.Stat(pathOutsideChroot(progressfile))
 
 	if err == nil {
 		// in progress
 		var content []byte
-		content, err = os.ReadFile(progressfile)
+		content, err = os.ReadFile(pathOutsideChroot(progressfile))
 		if err != nil {
 			return
 		}
 
 		progress := strings.TrimSpace(string(content))
 		r.Log.Info("Prep progress: " + progress)
 
-		if progress == "Completed" {
-			// If completed, update conditions and return doNotRequeue
+		if progress == "Failed" {
 			utils.SetStatusCondition(&ibu.Status.Conditions,
 				utils.GetCompletedConditionType(ranv1alpha1.Stages.Prep),
 				utils.ConditionReasons.Completed,
-				metav1.ConditionTrue,
-				"Prep completed",
+				metav1.ConditionFalse,
+				"Prep failed",
 				ibu.Generation)
 			utils.SetStatusCondition(&ibu.Status.Conditions,
 				utils.GetInProgressConditionType(ranv1alpha1.Stages.Prep),
 				utils.ConditionReasons.Completed,
 				metav1.ConditionFalse,
-				"Prep completed",
+				"Prep failed",
 				ibu.Generation)
-		} else if progress == "Failed" {
+		} else if progress == "completed-seed-image-pull" {
```

Comment:
Seems to make sense if we also update the in progress condition message with whatever in the progress file if it's not failed or completed case. We can leave it to the next PR though.